### PR TITLE
EDSC-4326: Update node and replace node-sass

### PR DESCRIPTION
**Pull request by trevorlang**
# Overview

### What is the feature?

Updates to a supported version of node and removes node-sass to fix installation issues related to its deprecation.

### What is the Solution?

- Uses node version `lts/jod`
- Removes `node-sass` in favor of `sass`

### What areas of the application does this impact?

While doing the change to `sass` I noticed the opportunity to fix some issues with the sass structure, example, and linting to clean up the project and improve dev UX.

# Testing

#### Check the example site locally
1. Run `npm run start`
2. Load the example site
3. Confirm all demo pages work as expected

#### Check the library works inside Earthdata Search
1. Run `npm pack` in the `edsc-timeline` directory
2. Install the generated package in `earthdata-search`
4. Spot check the timeline functionality
5. Run the unit and integration tests in `earthdata-seach`
6. Confirm there are no timeline related tests that fail

# Checklist

- [x] I have added automated tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by trevorlang — COMMENTED** [review]
Reminding myself to update edsc-eslint-config

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Before contributing to this project, please review [Contribution.md](https://github.com/nasa/earthdata-search/blob/master/CONTRIBUTING.md).
```
Comment (trevorlang) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Swap out links

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Please include relevant screenshots or files that would be helpful in reviewing and verifying this change.
+
+# Checklist
+
+- [ ] I have added automated tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
```
Comment (trevorlang) [inline .github/pull_request_template.md:33]: Add a step to talk about bumping node version. Check `edsc-geo`.

File: .nvmrc [hunk 3]
```diff
@@ -1 +1 @@
-lts/hydrogen
+lts/jod
```
Comment (trevorlang) [inline .nvmrc]: New line

File: cypress/e2e/focused_intervals.cy.js [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 import { getByTestId } from '../support/getByTestId'
 
-describe('Focused intervals', () => {
+describe('intervals', () => {
```
Comment (trevorlang) [inline cypress/e2e/focused_intervals.cy.js]: Add back "Focused"

File: example/src/components/Output/Output.scss [hunk 5]
```diff
@@ -0,0 +1,5 @@
+.output-item-entry {
+  &__heading {
+    width: 5rem;
+  }
+}
```
Comment (trevorlang) [inline example/src/components/Output/Output.scss]: New line

File: example/src/index.html [hunk 6]
```diff
@@ -1,16 +1,20 @@
...
+
 <body>
     <noscript>
         You need to enable JavaScript to run this app.
     </noscript>
     <div id="root"></div>
 </body>
-</html>
+
+</html>
```
Comment (trevorlang) [inline example/src/index.html]: Add new line

File: package.json
Comment (trevorlang) [inline package.json:1]: Replace maintainers

File: package.json [hunk 7]
```diff
@@ -89,14 +89,15 @@
...
     "nyc": "^15.1.0",
     "prop-types": "^15.8.1",
     "react": "^18.2.0",
+    "react-bootstrap": "^2.10.6",
     "react-dom": "^18.2.0",
     "react-github-corner": "^2.5.0",
+    "react-router-bootstrap": "^0.26.3",
     "react-router-dom": "^6.4.3",
-    "sass": "^1.51.0",
+    "sass": "1.77.6",
```
Comment (trevorlang) [inline package.json]: Dont pin this version

File: src/css/variables/_constants.scss [hunk 8]
```diff
@@ -0,0 +1,2 @@
+// Timeline
+$const__timeline--axis-height: 1.325rem;
```
Comment (trevorlang) [inline src/css/variables/_constants.scss]: New line

File: src/css/variables/_index.scss [hunk 9]
```diff
@@ -0,0 +1,2 @@
+@forward "colors";
+@forward "constants";
```
Comment (trevorlang) [inline src/css/variables/_index.scss]: New line

**Comment by eudoroolivares2016** [comment]
Using `npm pack ` and pulling this into EDSC and doing some regression testing I wasn't seeing any issues. I think those GH actions jobs there needs to be a config change into the settings though "the waiting for status hydrogen:) ones that is I ran into a similar issue in graphql but, don't have the permissions to update that one

**Review by eudoroolivares2016 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/custom.md [hunk 10]
```diff
@@ -0,0 +1,18 @@
+---
+name: Custom issue template
+about: Template for general tasks and improvements
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Please review [Contribution.md](https://github.com/nasa/earthdata-search/blob/master/CONTRIBUTING.md) before contributing to this project.
```
Comment (macrouch) [inline .github/ISSUE_TEMPLATE/custom.md]: This URL is still pointing to earthdata-search

**Review by macrouch — COMMENTED** [review]
Need to bump the npm version

**Review by macrouch — APPROVED** [review]

**Pull request merged**